src/SurveyDesign.jl

```diff
@@ -49,14 +48,15 @@ struct SurveyDesign <: AbstractSurveyDesign
     popsize::Symbol
     sampsize::Symbol
     strata::Symbol
-    pps::Bool
+    weights::Symbol # Effective weights in case of singlestage approx supported
```

Comment:
Do we need to have `weights` and `allprobs` given specifically as fields? We are not using them in any functions.

Reply:
we should keep consistent with the other 'pointers' stored as symbols (`popsize`, `sampsize` etc). I think weights should definitely be added, `allprobs` is there right now because `R` has it, so it is being used in some functions there.